app.js

```diff
@@ -65,7 +79,7 @@ app.get('/fetchTestCampaign/:id', async (req, res, next) => {
   }
 });
 
-app.get('/fetchTestCampaignMetaData/:id', async (req, res, next) => {
+app.get('/fetchTestMailCampaignMetaData/:id', async (req, res, next) => {
```

Comment:
de naam was vroeger al niet echt correct, sowieso is het woord "test" erin overbodig.
Dat mag er uit nu

Reply:
Ik neem deze mee in de refactor als dat goed is